docs/docs/testing-css-in-js.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Testing CSS-in-JS"
+---
+
+Popular CSS-in-JS libraries like [styled-components](https://github.com/styled-components/styled-components) or [emotion](https://github.com/emotion-js/emotion) can also be tested with the help of [jest-styled-components](https://github.com/styled-components/jest-styled-components) or [jest-emotion](https://github.com/emotion-js/emotion/tree/master/packages/jest-emotion) respectively. These packages improve Jest's built-in snapshot testing experience and are a great way to help avoid unintended changes to your website's UI. Please refer to your package's documentation to see if it also offers testing capabilities.
+
+*Snapshot serializers* like `jest-styled-components` or `jest-emotion` modify the standard output to a more meaningful and readable snapshot, e.g. by removing unnecessary information or displaying data in another format. Which ultimately leads to more compareable and effective snapshot tests.
```

Comment:
@m-allanson Is this a good explanation for snapshot serializers?

Reply:
What data formats are available? Which ones would you recommend?

Reply:
This is just a general statement unrelated to this specific document. It means that there are a lot of other snapshot serializers available with each formatting data in another suitable format/the format it was programmed for. Hence you can’t give recommendations